java/core/src/main/java/co/worklytics/psoxy/gateway/impl/GoogleCloudPlatformServiceAccountKeyAuthStrategy.java

```diff
@@ -0,0 +1,72 @@
+package co.worklytics.psoxy.gateway.impl;
+
+import co.worklytics.psoxy.gateway.ConfigService;
+import co.worklytics.psoxy.gateway.SourceAuthConfigProperty;
+import co.worklytics.psoxy.gateway.SourceAuthStrategy;
+import com.google.auth.Credentials;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.auth.oauth2.ServiceAccountCredentials;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.SneakyThrows;
+import lombok.extern.java.Log;
+
+import javax.inject.Inject;
+import java.io.ByteArrayInputStream;
+import java.util.*;
+import java.util.stream.Collectors;
+
+
+@Log
+@NoArgsConstructor(onConstructor_ = @Inject)
+public class GoogleCloudPlatformServiceAccountKeyAuthStrategy implements SourceAuthStrategy {
+
+    @Getter
+    private final String configIdentifier = "gcp_service_account_key";
```

Comment:
It will always have same value, so I'd put it static too:

```suggestion
    private final static String configIdentifier = "gcp_service_account_key";
```